### DISCUSSION TITLE
Active Links in Menu

### DISCUSSION THREAD

**Discussion by harshitv804** [athul/archie#89] [Ideas]
Please add a feature for highlighting active Links in Menu same as the hover color. I am just an amateur, please help. Also, I really want to appreciate your theme, it's so cool. 

Thanks!

**Comment by webyang-male** [comment]
> Please add a feature for highlighting active Links in Menu same as the hover color. I am just an amateur, please help. Also, I really want to appreciate your theme, it's so cool.
> 
> Thanks!
在a标签上增加hover时的css样式处理就行了

![image](https://github.com/user-attachments/assets/2f32278b-d3bf-4639-927b-8103e86d0b0b)
